docs/04-operation-guides/troubleshooting/apix-03-404-not-found.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cannot connect to a service exposed by an API Rule - 404 Not Found
+---
+
+## Symptom
+
+You reach your service and get `404 Not Found` in response.
+
+## Remedy
+
+In such case, make sure that:
+
+- Proper Oathkeeper Rule has been created:
+
+  ```bash
+  kubectl get rules.oathkeeper.ory.sh -n {NAMESPACE}
+  ```
+
+  >**TIP:** Name of the Rule consists of the name of the API Rule and a random suffix.
+
+- Proper VirtualService has been created:
+
+  ```bash
+  kubectl get virtualservices.networking.istio.io -n {NAMESPACE}
+  ```
+
+  >**TIP:** Name of the VirtualService consists of the name of the API Rule and a random suffix.
+
+Sometimes Oathkeeper Maester controller stops reconciling Rules on long-living clusters. This can result in random `404 Not Found` responses, because Oathkeeper does not contain Rules reflecting the actual state of the cluster. A simple restart of the Pod resolves the issue, but you might want to verify if that is the issue you have encountered. To do so:
```

Comment:
```suggestion
Sometimes Oathkeeper Maester controller stops reconciling Rules on long-living clusters. This can result in random `404 Not Found` responses because Oathkeeper does not contain Rules reflecting the actual state of the cluster. A simple restart of the Pod resolves the issue, but you might want to verify if that is the issue you have encountered. To do so, follow these steps:
```